src/components/grid-view/GridView.scss

```diff
@@ -4,30 +4,17 @@
 .bdl-GridView {
     padding: 0 5px;
 
-    &.bdl-GridView--columns-1 .bdl-GridView-slot {
-        $columns1prct: 100%;
-
-        flex-basis: $columns1prct;
-        max-width: $columns1prct;
+    .ReactVirtualized__Table__rowColumn:first-of-type {
```

Comment:
It's a little confusing, but :first-of-type refers to the element rather than the class. Can you use :first-child, instead?